tests/generators/stacked_foreach.bzz

```diff
@@ -0,0 +1,27 @@
+// INTEGRATION TESTS IGNORE
+czynność główna() {
+    przekaźnik liczby_naturalne() -> Liczba {
+        zm x: Liczba = 0
+        dopóki(prawda) {
+            przekaż x
+            x = x+1
+        }
+    }
+
+    zm wynik: Liczba = 0
+    zm m: Liczba = wczytaj()
+    otrzymując x: Liczba od liczby_naturalne() {
+        jeśli(x > m) przerwij
+        otrzymując y: Liczba od liczby_naturalne() {
+            jeśli(y > m) przerwij
+            otrzymując z: Liczba od przedziału(m+1) {
+                otrzymując w: Liczba od liczby_naturalne() {
+                    jeśli(w > m) przerwij
+                    wynik = wynik + x*y*w*(z+1)
+                }
+            }
+        }
+    }
+    //(1+2+3+..+m)^4
```

Comment:
I didn't check the formula behind this code, but I assume you got it right :smile:

Reply:
Actually, it's `1*nested_loop_result + 2*nested_loop_result + ... = (1+2+..+m)*nested_loop_result` :smiley: